docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -283,41 +314,71 @@ When the `CONSTRAINTS` option doesn't contain `count`, `REPLICAS` must be specif
 
```

Comment:
Could you please also check the old row nr 274-282?
Old row nr 278
```
When all the replica counts are specified in the `CONSTRAINTS` option, `REPLICAS` can also be omitted. For example, `CONSTRAINTS="{+zone=bj:2,+zone=sh:1}", ROLE=voter` indicates that the `REPLICAS` is 3.
```
Again "indicates that the `REPLICAS` is 3" => "sets the `REPLICAS` to 3" ?
And why the comma in between `CONSTRAINTS` and `ROLE`?

Old row nr 280
```
For example, in the case `CONSTRAINTS="{+zone=bj:2,+zone=sh:1}", ROLE=voter, REPLICAS=4`, 2 replicas are in `bj` and 1 in `sh`, and the last replica can be anywhere, including `bj` and `sh`.
```
Does that mean we would need a '-zone=tw:0' in the CONSTRAINTS to *not* have any replica in tw zone?

Reply:
> Does that mean we would need a '-zone=tw:0' in the CONSTRAINTS to not have any replica in tw zone?

True.

Reply:
I have fixed the syntax, so I think this point is resolved? Each of the options are now `table_options` without comments.

The behavior of REPLICAS (now `FOLLOWERS`, `LEADERS`, or `VOTERS` directly) has also changed so that the constraint doesn't adjust the count even if it detects it is possible. Rather, if it is not possible based on the count, you will now receive an error (see just below).